front/src/modules/ui/dropdown/hooks/useDropdownButton.ts

```diff
@@ -11,6 +17,36 @@ export const useDropdownButton = ({ dropdownId }: { dropdownId: string }) => {
     goBackToPreviousHotkeyScope,
   } = usePreviousHotkeyScope();
 
+  const { ViewBarRecoilScopeContext } = useViewBarContext();
```

Comment:
This is not the right place for this because useDropdownButton shouldn't be aware of where it's used.

Reply:
Reset state should happen in each component that use DropdownButton, by adding an handleDropdownButtonClose handler and calling resetState inside it.